src/autocomplete/dataset.js

```diff
@@ -93,7 +93,7 @@ _.mixin(Dataset.prototype, EventEmitter, {
     var renderArgs = [].slice.call(arguments, 2);
     this.$el.empty();
 
-    hasSuggestions = suggestions && suggestions.length;
+    hasSuggestions = suggestions && suggestions.length > 0;
```

Comment:
I did some investigation and it seems the non angular build is behaving not the way it should given the code. When you have two datasets and one sends an empty suggestion list then both datasets ends up as being "not empty" while in angular one of them ends up as being empty (as planned) which results in the whole autocomplete being hidden.

I am leaving for vacation but dig further as you wish. This current fix you did is not fixing the issue for me